src/sentry/snuba/metrics/extraction.py

```diff
@@ -779,7 +779,7 @@ def _is_standard_metrics_field(field: str) -> bool:
 
 
 def _is_error_field(token: str) -> bool:
-    return token.startswith("error.")
+    return token.startswith("error.") or token.startswith("level")
```

Comment:
Without this, `_get_query_supported_by(query)` would say that the query `level:foo` is an on-demand supported metric and `should_use_on_demand_metrics` would return `True`.
https://github.com/getsentry/sentry/blob/2b021cc5f45e3bbe6b46573abacf39a473eb8e5e/src/sentry/snuba/metrics/extraction.py#L589